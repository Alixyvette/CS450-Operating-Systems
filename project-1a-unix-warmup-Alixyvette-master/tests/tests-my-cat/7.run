./my-cat INPUT_DIR/7a.in INPUT_DIR/7b.in INPUT_DIR/7c.in INPUT_DIR/7d.in
