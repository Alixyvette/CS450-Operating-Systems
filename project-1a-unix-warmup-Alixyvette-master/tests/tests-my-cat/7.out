this one has stuff in it
this one does too
my-cat: cannot open file
